release-content/0.14/release-notes/13450_Implement_PBR_anisotropy_per_KHR_materials_anisotropy.md

```diff
@@ -1,4 +1,26 @@
 <!-- Implement PBR anisotropy per `KHR_materials_anisotropy`. -->
 <!-- https://github.com/bevyengine/bevy/pull/13450 -->
 
-<!-- TODO -->
+[Anisotropic materials](https://en.wikipedia.org/wiki/Anisotropy) change based on the axis of motion, such as how wood behaves very differently when working with versus against the grain.
+But in the context of phsically-based rendering, **anisotropy** refers specifically to a feature that allows roughness to vary along the tangent and bitangent directions of a mesh.
+In effect, this causes the specular light to stretch out into lines instead of a round lobe. This is useful for modeling brushed metal, hair, and similar surfaces.
+Support for anisotropy is a common feature in major game and graphics engines; Unity, Unreal, Godot, three.js, and Blender all support it to varying degrees.
+
+{{ compare_slider(
+    left_title="Without Anisotropy",
+    left_image="without_anisotropy.png",
+    right_title="With Anisotropy",
+    right_image="with_anisotropy.png"
+) }}
+
+Two new parameters have been added to [`StandardMaterial`](https://dev-docs.bevyengine.org/bevy/pbr/struct.StandardMaterial.html): `anisotropy_strength` and `anisotropy_rotation`.
```

Comment:
We'll have to remember to update this link.

Reply:
Yep, planning a find-and-replace pass at the end.